app/javascript/v3/components/Form/ProfileAvatar.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    class="relative rounded-xl h-[72px] w-[72px] cursor-pointer"
+    @mouseover="onImageOver"
+    @mouseleave="onImageOut"
```

Comment:
@muhsin-k Let us use the `group` and `group-hover` classes from the tailwind to do these mouse over actions. [ref](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state)